docs/GettingStartedDocs/install_oe_sdk-Simulation.md

```diff
@@ -0,0 +1,27 @@
+# Install the Open Enclave SDK (non-SGX Ubuntu 16.04/18.04)
+
+## Considerations
+The Open Enclave SDK can be installed on non-SGX Ubuntu 16.04 and 18.04 systems. It can also be used in simulation mode, although some features may not work. Specifically these features are not supported when running in simulation mode:
+- Enclave signature/attribute verification
```

Comment:
It is not clear what _attribute verification_ means.
_Enclave signing and measurement_ sounds better.

Reply:
@johnkord Can you double check if signing needs to be run in SGX machines?

Reply:
@anakrish What do you mean by "if signing needs to be run"? You mean, do enclaves have to be signed before they are created/initialized?

Reply:
Oh, do you mean specifically: Can enclaves be signed on non-SGX systems? I believe so. My non-SGX Ubuntu 16.04 just built and signed an enclave.

Reply:
Number 5 is incorrect. Before private preview, all our enclaves were signed and ran well in simulation mode.